### ISSUE TITLE
For Open Ownership, unflatten needs to put statements in order.

### ISSUE THREAD

**Issue by odscjames** [OpenDataServices/flatten-tool#228]
From #223, @timgdavies pointed out: 

> Each of the entityStatements or personStatements referenced by a particular ownershipOrControlStatement MUST appear before that particular statement in the ordered array.

I believe this can be sorted with some careful use of filenames but 

a) test that
b) make sure it's documented on Open Ownerships page in the docs

**Comment by odscjames** [comment]
Yeah, adding numbers so the files are loaded in order totally works:

*  1entityStatement.csv
* 1entityStatement-identifiers.csv
* 2personStatement-addresses.csv
* 2personStatement.csv
* 2personStatement-names.csv
* 2personStatement-nationalities.csv
* 3ownershipOrControlStatement.csv
* 3ownershipOrControlStatement-interests.csv

Make sure this is documented on Open Ownerships page in the docs

**Comment by robredpath** [comment]
I don't love this. 

Would it be hard to either allow the user to specify ordering (maybe specifying prefix order?) or to hard-code (and in the future pick up conf from a file) the BODS-specific ordering?

**Comment by odscjames** [comment]
I don't really either. I was going to start by making sure the tool was at least minimally useable ASAP (hence these docs). 

Then I was going to discuss things like you suggest, but with the assumption that hard-coding anything too specific about one standard into the tool was going to be problematic - so things like config files are probably the way to go.
